sc-platform-api/src/documentation/user/updateUser.ts

```diff
@@ -0,0 +1,37 @@
+import { ApiDocumentationParams } from '../decorators';
+import { UpdateUserDTO, UserResponse } from '@student-council-platform/utils';
+
+export const UserDocumentationUpdateUser: ApiDocumentationParams = {
+  isAuth: true,
+  ok: {
+    type: UserResponse,
+  },
+  badRequest: {
+    description: `\n
+    InvalidBodyException:
+      Username must be a string
+      Username is not correct (a-zA-Z0-9_), or too short (min: 2), or too long (max: 40)
+      The password must be between 6 and 32 characters long, include at least 1 digit and 1 latin letter
+      Password must be a string
+      First name must be a string
+      First name is not correct (ҐЄІЇЬА-ЩЮЯґєіїьа-щюя)
+      Last name must be a string
+      Last name is not correct (ҐЄІЇЬА-ЩЮЯґєіїьа-щюя)
+      Middle name must be a string
+      Middle name is not correct (ҐЄІЇЬА-ЩЮЯґєіїьа-щюя)
+      Email must be a string
+      Email must be an email
+    `,
+  },
+  unauthorized: true,
```

Comment:
In this endpoint forbidden response is also possible (in `AdminOrMeGuard`)
```suggestion
  unauthorized: true,
  forbidden: true,
```